### PR TITLE
Feature/cindy

### PR DESCRIPTION
**Pull request by cindyfeldman**
Json file parsed

### CODE REVIEW

**Review by ChaoticLeah — CHANGES_REQUESTED** [review]
Try to make quesFile a non-direct path. I would suggest trying something like "./deforestation-game/Assets/Questions/Questions.json"

**Comment by buttercakeee** [comment]
I just agree with Kevin's suggestion and other than that it's all good.

**Comment by adigudi** [comment]
> I just agree with Kevin's suggestion and other than that it's all good.

Same here

File: Assets/Scripts/QuestionButtonAmt.cs [hunk 1]
```diff
@@ -9,43 +10,95 @@ public class QuestionButtonAmt : MonoBehaviour
...
+            for (int i = 0; i <; i++)
+            {
+                answer = items.Questions[i].Answers[i].AnswerText;
+                Debug.Log(items.Questions[i].Answers[i].AnswerText);
+            }
+        }
+        answerText.text = answer;
 
-    void setQuestion(Text questionText, String question)
+    }*/
```
Comment (kronodeus) [inline Assets/Scripts/QuestionButtonAmt.cs:63]: We should leave out this commented code

**Review by ChaoticLeah — APPROVED** [review]
It seems good from the code.

**Pull request merged**